test/index.js

```diff
@@ -32,7 +32,6 @@ tests.miscellaneous(factory, { skip: ['dns', 'resolve'] })
 tests.pubsub(factory)
 // these are rarely flaky
 tests.swarm(factory)
-tests.miscellaneous(factory, { skip: ['dns', 'resolve'] })
 
```

Comment:
The diff doesn't show but the duplicate is up on line 17

Reply:
You beat me to it! :)

Reply:
I knew you'd like that :D